fpdf/drawing.py

```diff
@@ -1264,6 +1266,16 @@ def fill_opacity(self, new):
 
         super().__setattr__(PDFStyleKeys.FILL_ALPHA.value, new)
 
+    @property
+    def fill_pattern(self):
+        """The desired fill pattern for this path/group."""
+        pass
+
+    @fill_pattern.setter
+    def fill_pattern(self, new):
+        """The desired fill pattern for this path/group."""
+        pass
+
```

Comment:
Those getter / setter do nothing, are in fact not really needed if the property you introduced is public (no leading underscore)